SDK v2 Examples/quickstart/quickstart/ViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ViewController.swift
+//  quickstart
+//
+//  Created by Youel Kaisar on 2020-09-02.
+//  Copyright © 2020 Mappedin. All rights reserved.
+//
+
+import UIKit
+import Mappedin
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Create a Mappedin object and enter your keys through the initializer
+        let mappedIn = Mappedin(mappedinKey: "5f2c5b9aa5fdf5001a6b971d", mappedinSecret: "tsKYDtd9xnks2WNu9YHS5SrfHwwENGeHnaAyMTI5jPhpsEbU")
```

Comment:
These keys need to change to:
Mappedin(mappedinKey: "5f4e59bb91b055001a68e9d9", mappedinSecret: "gmwQbwuNv7cvDYggcYl4cMa5c7n0vh4vqNQEkoyLRuJ4vU42")

I'll invalidate the keys above.

Reply:
I got those from the quickstart